.github/workflows/ae-ano-detection.yml

```diff
@@ -29,16 +29,16 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v3
 
-#      - name: Run ML experiment with DVC
-#        uses: ./.github/actions/run-ml-experiment-with-dvc
-#        with:
-#          ml-experiment-path: ./services/ae-ano-detection/model_creation
-#          aws-access-key-id: ${{ secrets.ae-ano-detection_AWS_ACCESS_KEY_ID }}
-#          aws-secret-access-key: ${{ secrets.ae-ano-detection_AWS_SECRET_ACCESS_KEY }}
-#
-#      - name: Move the model to model_serving directory
-#        shell: bash
-#        run: mv ./services/ae-ano-detection/{model_creation,model_serving}/mnist_model.h5
+      - name: Run ML experiment with DVC
+        uses: ./.github/actions/run-ml-experiment-with-dvc
+        with:
+          ml-experiment-path: ./services/ae-ano-detection/model_creation
+          aws-access-key-id: ${{ secrets.DIGIT_RECOGNITION_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.DIGIT_RECOGNITION_AWS_SECRET_ACCESS_KEY }}
```

Comment:
You should create your own set of environment variables for your service (`AE_ANO_DETECTION_AWS_ACCESS_KEY_ID` and `AE_ANO_DETECTION_AWS_SECRET_ACCESS_KEY`) even if the values are the same as the ones set in the `DIGIT_RECOGNITION_AWS_ACCESS_KEY_ID` and `DIGIT_RECOGNITION_AWS_SECRET_ACCESS_KEY` variables. This could allow the changes of individual services to others S3 servers without breaking existing services.

You can check the [_How to create a GitHub Secret_](https://csia-pme.github.io/csia-pme/how-to-guides/how-to-create-a-github-secret/) guide to create your own secrets for your project.